spec/sass_3_5/change-equality-of-lists/input.sass

```diff
@@ -0,0 +1,5 @@
+a
```

Comment:
This should probably be using SCSS syntax. The only indented syntax tests are under https://github.com/sass/sass-spec/tree/master/spec/sass although I get why that is confusing since this folder does start with `sass`